supra-cli/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+// use libp2p::PeerId;
+pub use sc_cli::{CliConfiguration, SharedParams};
+use structopt::StructOpt;
+use bs58::{decode, Alphabet};
+use std::str::FromStr;
+
+#[derive(Debug, StructOpt)]
+pub struct SupraCmd {
+    #[allow(missing_docs)]
+	#[structopt(flatten)]
+	pub shared_params: SharedParams,
```

Comment:
`shared_params` and the corresponding `impl CliConfiguration for PeerIdHexCmd` can be removed since they are not used.